<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CamMosLoggingWidget</class>
 <widget class="QWidget" name="C_CamMosLoggingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_CamMosLoggingWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_CamOgeWiSettingSubSection" name="pc_WiHeader" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="pc_WiContent" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="3" column="0">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabFormat">
          <property name="text">
           <string>Format TR</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="stw_opensyde_gui_elements::C_OgeLeDark" name="pc_LeFile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabFile">
          <property name="text">
           <string>File TR</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="stw_opensyde_gui_elements::C_CamOgePubDarkBrowse" name="pc_PubBrowse">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="stw_opensyde_gui_elements::C_CamOgeLeDarkBrowse" name="pc_LeFolder">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="stw_opensyde_gui_elements::C_CamOgeCbxDark" name="pc_CbxFormat">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="stw_opensyde_gui_elements::C_CamOgeCbxDark" name="pc_CbxOverwrite">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabOverwrite">
          <property name="text">
           <string>Overwrite Mode TR</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabFolder">
          <property name="text">
           <string>Folder TR</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_CamOgeWiSettingSubSection</class>
   <extends>QWidget</extends>
   <header>C_CamOgeWiSettingSubSection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLeDark</class>
   <extends>QLineEdit</extends>
   <header>C_OgeLeDark.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_CamOgeCbxDark</class>
   <extends>QComboBox</extends>
   <header>C_CamOgeCbxDark.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_CamOgeLeDarkBrowse</class>
   <extends>QLineEdit</extends>
   <header>C_CamOgeLeDarkBrowse.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_CamOgePubDarkBrowse</class>
   <extends>QPushButton</extends>
   <header>C_CamOgePubDarkBrowse.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pc_LeFolder</tabstop>
  <tabstop>pc_PubBrowse</tabstop>
  <tabstop>pc_LeFile</tabstop>
  <tabstop>pc_CbxFormat</tabstop>
  <tabstop>pc_CbxOverwrite</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
